src/statistics.sh

```diff
@@ -17,82 +17,83 @@ declare -gA shared_data=(["deploy"]='' ["build"]='' ["list"]='' ["uninstall"]=''
 
 function statistics()
 {
-  local info_request="$1"
-  local day=$(get_today_info '+%d')
-  local year_month_dir=$(get_today_info '+%Y/%m')
-  local date_param
+  local target="$1"
 
   if [[ "$1" == -h ]]; then
     statistics_help
     exit 0
   fi
 
   shift 1 # Remove the first option, i.e., --day, --week, or --year
-  local date_param="$@"
 
-  if [[ ${configurations[disable_statistics_data_track]} == 'yes' ]]; then
+  if [[ "${configurations[disable_statistics_data_track]}" == 'yes' ]]; then
     say "You have disable_statistics_data_track marked as 'yes'"
     say "If you want to see the statistics, change this option to 'no'"
     return
   fi
 
   # Default to day
-  if [[ -z "$info_request" ]]; then
-    info_request="--day"
+  if [[ -z "$target" ]]; then
+    target='--day'
   fi
 
-  if [[ -z "$date_param" ]]; then
-    date_param=$(get_today_info '+%Y/%m/%d')
-  fi
-
-  case "$info_request" in
+  case "$target" in
     --day)
-      if [[ -z "$date_param" ]]; then
-        target_day="$KW_DATA_DIR/statistics/$year_month_dir/$day"
-      else
-        target_day="$KW_DATA_DIR/statistics/$(date_to_format "$date_param")"
+      local day
```

Comment:
@rodrigosiqueira, do we want to keep the `local` declarations all together in the beginning of the function? you mentioned something like this

Reply:
Yeah... let's do that since this can avoid shellcheck problems.